wiki/Announcement_messages/de.md

```diff
@@ -0,0 +1,57 @@
+---
+tags:
+  - announce
+  - announce usergroup
+  - announce user group
+  - Ankündigung
+  - Ankündigungsgruppe
+---
+
+# Ankündigungsnachrichten
+
+![Ankündigungsbenachrichtigung](img/notification.jpg "Eine Benachrichtigung über eine Ankündigungsnachricht")
+
+Eine **Ankündigungsnachricht** ist eine besondere Nachricht, die an mehrere Nutzer gleichzeitig versendet wird.
+Die Hauptunterschiede zwischen Ankündigungsnachrichten und normalen Nachrichten sind:
+
+- Die maximale Länge von 1024 statt 450 Zeichen
+- Markdown-Syntax[^note-images] kann zum Formatieren einer Ankündigungsnachricht verwendet werden
+- Gleichzeitige Zustellung an mehrere Nutzer
+- Die Möglichkeit, die Einstellung `Blockiere Nachrichten von Benutzern, die nicht in deiner Freundesliste sind` zu umgehen
+- Nur Nutzer, die Ankündigungsnachrichten versenden dürfen, können auf Ankündigungsnachrichten antworten
+
+## Berechtigung
+
+Um Ankündigungsnachrichten über die Webseite versenden zu dürfen, muss der jeweilige Nutzer Mitglied des [Globalen Moderationsteams](/wiki/People/Global_Moderation_Team), des [Nomination Assessment Teams](/wiki/People/Nomination_Assessment_Team) oder der Ankündigungsgruppe[^note-group] sein.
+Allerdings dürfen nur Mitglieder der Ankündigungsgruppe Ankündigungen im Chat über die [osu!api v2](https://osu.ppy.sh/docs/index.html#create-channel) versenden.
+
+### Einen Beitrittsantrag stellen
+
+Nutzer können einen Antrag zum Beitritt der Ankündigungsgruppe stellen, indem sie eine E-Mail mit dem Betreff `Announce Usergroup Request` an [accounts@ppy.sh](mailto:accounts@ppy.sh) senden. Dazu muss dieselbe E-Mail-Adresse verwendet werden, die mit dem osu!-Account des Nutzers verbunden ist.
+
+Der Textkörper der E-Mail sollte Folgendes beinhalten:
+
+- Den osu!-Nutzernamen
+- Eine kurze Erklärung, wofür die Berechtigung zum Versenden von Ankündigungsnachrichten benötigt wird
+- Wie häufig Ankündigungsnachrichten versendet werden sollen
+
+Das [Account-Support-Team](/wiki/People/Account_support_team) wird den Antrag prüfen und mit ihrer Entscheidung antworten.
```

Comment:
For your information, if `you` was present in an article, I would have adopted this into my translation in the past. The way this article is written, I'm not sure why `you` is included here in the English part. Preferring to keep `you`s out of this article, I really like the way you translated this part here.

See [Tournaments/Official support § Eligibility](https://osu.ppy.sh/wiki/de/Tournaments/Official_support#eligibility) as a comparison.

Reply:
Maybe the english article needs a rewrite first.. IIRC, translations shouldn't correct things that are wrong in the english version. Good catch, I don't know why I glossed over this.

Reply:
I opened #10172 to address this in the english version. If this PR is merged after 10172, there will be no disparities.

Reply:
It's merged.